onnxruntime/python/tools/transformers/dev_benchmark.cmd

```diff
@@ -21,12 +19,12 @@ set run_torchscript=false
 
 REM Devices to test.
 REM Attention: You cannot run both CPU and GPU at the same time: gpu need onnxruntime-gpu, and CPU need onnxruntime.
-set run_gpu_fp32=false
-set run_gpu_fp16=false
-set run_cpu_fp32=true
-set run_cpu_int8=true
+set run_gpu_fp32=true
+set run_gpu_fp16=true
+set run_cpu_fp32=false
+set run_cpu_int8=false
```

Comment:
Should we include INT4 CPU and INT4 GPU as benchmark options?

Reply:
Yes, maybe later.